Dockerfile-debug

```diff
@@ -1,19 +1,15 @@
-FROM debian:bookworm-slim
-RUN apt-get update && apt-get install -y --no-install-recommends \
+FROM alpine:3.20.1
+RUN apk add \
```

Comment:
We could also run `apk update && apk upgrade` before installing any dependencies to ensure the pkg lists are up-to-date and we receive the latest versions.